src/iuse.cpp

```diff
@@ -3240,35 +3240,38 @@ int iuse::granade_act(player *, item *it, bool t, const tripoint &pos)
             case 2:
                 sounds::sound(pos, 100, _("BUFFS!!"));
                 g->draw_explosion( pos, explosion_radius, c_green );
-                for( const tripoint &dest : g->m.points_in_radius( pos, explosion_radius ) ) {
```

Comment:
You need a merge or rebase, this reverts part of a recently merged PR.
Just merge and take the version in HEAD for this hunk of code.